packages/components/psammead-radio-schedule/src/index.test.jsx

```diff
@@ -17,4 +19,16 @@ describe('RadioSchedule', () => {
       selectedService: 'arabic',
     }),
   );
+
+  it('should render with passed component', () => {
+    const { container } = render(
+      renderRadioSchedule({
+        linkComponent: 'aside',
```

Comment:
What made you pick an `aside` element with a `to` attr? Just to prove it'll have a go at rendering whatever you give it?

Reply:
We wanted to pick an element not likely to be used in immediate future updates to the radio schedules component. That's why we picked `aside`. And we picked `to` attr to ensure that other allowed JSX attributes other than the `href` default work

Reply:
Yea I didn't think 'to' would be a valid attribute for an aside but #yolo 

Oh - it looks like since React 16, they allow anything (from https://reactjs.org/blog/2017/09/08/dom-attributes-in-react-16.html):

```
// Your code:
<div mycustomattribute="something" />
```
would render an empty div to the DOM with React 15:

```
// React 15 output:
<div />
```

In React 16, we are making a change. Now, any unknown attributes will end up in the DOM:
```
// React 16 output:
<div mycustomattribute="something" />
```